ingest_data/sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -2,15 +2,16 @@
  "cells": [
```

Comment:
<p>Grammar nits: remove all uses of "will", "would", "we'll", etc, and just use present tense throughout. I.e. <code>customer would receive</code> -&gt; <code>customer receives</code> .</p><p> </p><p>Formatting nit: recommended to use plain quotes (<code>""</code>) instead of fancy quotes (<code>“”</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Fixed — removed all "will", "would", "we'll", and fancy quotes